### ISSUE TITLE
The predicted output remains empty during training. Training loss is decreasing but validation loss is constant

### ISSUE THREAD

**Issue by thani-ath-nain** [NVIDIA/NeMo#5702]
**Describe the bug**

I am training stt_en_conformer_ctc_large on my custom datset+librispeech dev and test splits. Currently, half of epoch 0 has completed but the best value of WER is 1 and logs show epty in predicted. I am a beginner and do not understand the ASR process fully so it must be something that I am doing wrong. Can someone help me?. I am tokenizing using SPE with a vocab size of 15000, and am using the default training script from
```
https://github.com/NVIDIA/NeMo/blob/main/examples/asr/asr_ctc/speech_to_text_ctc_bpe.py
```
and the config from 
```
https://github.com/NVIDIA/NeMo/blob/main/examples/asr/conf/conformer/conformer_ctc_bpe.yaml
``` 
with some fields changed according to my need

**Steps/Code to reproduce bug**

Running the above training script with the config file which contains all the relevant path information.

**Expected behavior**

At least something should be predicted in logs

**Environment overview (please complete the following information)**

 - Environment location: AWS
 - Method of NeMo install: [pip install nemo_toolkit["all"]

**Environment details**

If NVIDIA docker image is used you don't need to specify these.
Otherwise, please provide:
- OS version Ubuntu 20.04
- PyTorch version 11.7
- Python version 3.8

**Additional context**

Tesla T4

**Comment by thani-ath-nain** [comment]
Looking around, it seems even though other people have had this issue or its variants none found a solution. If it helps, I am training on an English dataset and reference is printing flawlessly.

**Comment by thani-ath-nain** [comment]
Now I have tried different tokenizers including spe(unigram, word, char) and wpe. But still the output is same. Model does pick up the tokenizer with num_classes being set to total tokens inside vocab.txt but predicted remains empty. Running inference on trained checkpoints also gives empty strings as output.

**Comment by titu1994** [comment]
When you train from scratch, or chance the decoder, your model has to learn to align speech and text all from scratch. It will take far more than half an epoch to learn this. Let the training continue for several dozen epochs and use a min batch size of 128 (with gradient accumulation).

**Comment by thani-ath-nain** [comment]
Thank you for your reply,
